src/index.ts

```diff
@@ -7,46 +7,46 @@
  * without explicit written authorization from Numenta Inc.
  */
 import * as core from "@actions/core";
+import { DynamoDBClient, GetItemCommand, GetItemInput } from "@aws-sdk/client-dynamodb";
+import { fromEnv } from "@aws-sdk/credential-providers";
+import { marshall } from "@aws-sdk/util-dynamodb";
 import { Octokit } from "octokit";
 
 const CLA_LINK = "https://github.com/thousandbrainsproject/cla/issues/new?template=01_sign_cla.yml&labels=signature+CLA+v1&title=Contributor+License+Agreement+v1+Signature";
+const CLA_SIGNATORIES_TABLE_NAME = "DynamoDB-CLASignatoriesTable-12GCQ0F0TB473";
```

Comment:
non-blocking:

These constants might benefit from being environment variables:
- it would allow other people to fork this repo and use it themselves with no code changes.
- they might change at some point in time

Reply:
I agree that would make things more portable. I'll defer as future work for now.